### PR TITLE
fix: fix bugs in Cashbox history

### PR DESCRIPTION
**Pull request by ubavic**

### CODE REVIEW

**Comment by ubavic** [comment]
[Cashbox history isn't displaying any history](https://trello.com/c/l8WjAy3e/1060-cashbox-history-isnt-displaying-any-history)

File: new-lamassu-admin/src/pages/Maintenance/CashboxHistory.js [hunk 1]
```diff
@@ -111,21 +110,27 @@ const CashboxHistory = ({ machines, currency }) => {
...
+    const _performedBy = R.prop(
+      'performedBy',
+      R.find(f => f.id === row.id, fields)
+    )
+
+    const performedBy = _performedBy === '' ? null : _performedBy
+
     schema
-      .isValid(fields)
+      .isValid(performedBy)
```
Comment (siiky) [inline new-lamassu-admin/src/pages/Maintenance/CashboxHistory.js]: Looking at the `schema`, I don't understand why this is working... From my understanding, the variable given to `isValid()` should be an object with the key `"performedBy"`.
But again, it's working on release too, so maybe it's my misunderstanding.

File: new-lamassu-admin/src/pages/Maintenance/CashboxHistory.js [hunk 2]
```diff
@@ -192,16 +197,25 @@ const CashboxHistory = ({ machines, currency }) => {
       width: 180,
       textAlign: 'left',
       view: it => {
-        if (!editing)
+        if (!R.any(R.propEq('id', it.id), fields))
```
Comment (siiky) [inline new-lamassu-admin/src/pages/Maintenance/CashboxHistory.js]: You could define `editing` as a function over `it`, to be able to reuse below.

**Review by siiky — COMMENTED** [review]
WDYT?

It cleanly applied to release as well, and it worked like a charm there!

**Review by siiky — APPROVED** [review]
LGTM! It applied cleanly on release again. Thanks

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**